src/extent.c

```diff
@@ -216,7 +216,10 @@ extents_insert_locked(tsdn_t *tsdn, extents_t *extents, extent_t *extent,
 		extent_list_append(&extents->lru, extent);
 	}
 	size_t npages = size >> LG_PAGE;
-	atomic_add_zu(&extents->npages, npages);
+	size_t cur_extents_npages =
+	    atomic_load_zu(&extents->npages, ATOMIC_RELAXED);
+	atomic_store_zu(&extents->npages, cur_extents_npages + npages,
```

Comment:
As discussed on [gitter](https://gitter.im/jemalloc/jemalloc?at=58c0944b7ceae5376a8e6c8d), comments just above the atomic update code blocks will help to clarify why this is okay (mutex protection).